parse.go

```diff
@@ -55,9 +56,17 @@ var (
 	ErrorMismatchPixelDataLength = errors.New("the size calculated from DICOM elements and the PixelData element's VL are mismatched")
 )
 
+func Parse(in io.Reader, bytesToRead int64, frameChan chan *frame.Frame, opts ...ParseOption) (Dataset, error) {
+	return parseInternal(in, bytesToRead, frameChan, opts...)
+}
+
+func ParseUntilEOF(in io.Reader, frameChan chan *frame.Frame, opts ...ParseOption) (Dataset, error) {
+	return parseInternal(in, LimitReadUntilEOF, frameChan, opts...)
```

Comment:
we should be passing the exported const from the `dicomio` package e.g. we should be able to do 

```suggestion
	return parseInternal(in, dicomio.LimitReadUntilEOF, frameChan, opts...)
```